packages/react/src/components/Forms/Examples/ReactHookForm/Sports.stories.tsx

```diff
@@ -46,6 +50,10 @@ export const Sports = () => {
     resolver: yupResolver(sportsSchema),
   });
 
+  useEffect(() => {
```

Comment:
What is the purpose of this?

Reply:
This usecase came from richie.
We fetch some data, here address data and when they arrive we do a `reset(address)` that initialize the form with our address.